plugins/database/hana/hana_test.go

```diff
@@ -0,0 +1,172 @@
+package hana
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/vault/builtin/logical/database/dbplugin"
+	"github.com/hashicorp/vault/plugins/helper/database/connutil"
+)
+
+var (
+	testHANAImagePull sync.Once
+)
+
+func TestHANA_Initialize(t *testing.T) {
+	if os.Getenv("HANA_URL") == "" || os.Getenv("VAULT_ACC") != "1" {
+		return
```

Comment:
I think `t.SkipNow()` would be more appropriate rather than assume the test passes. Same with the other test cases.